### PR TITLE
bump gems

### PR DESCRIPTION
**Pull request by andyborn**

### CODE REVIEW

**Comment by dim** [comment]
More to it, pipeline is broken + you need to add Ruby 3.2

**Comment by andyborn** [comment]
> More to it, pipeline is broken + you need to add Ruby 3.2



> More to it, pipeline is broken + you need to add Ruby 3.2

have fixed the rubocop errors but will need review.  how do I add Ruby 3.2?

File: Gemfile [hunk 1]
```diff
@@ -1,2 +1,10 @@
 source 'http://rubygems.org'
 gemspec
+
+gem 'rake'
+gem 'sqlite3'
+
+group 'development', 'test' do
```
Comment (dim) [inline Gemfile:7]: you don't need to group them in libs

**Review by dim — COMMENTED** [review]
You can add 3.2 to the matrix in `.github/workflows/test.yaml`

**Pull request merged**